xinference/core/model.py

```diff
@@ -168,6 +171,11 @@ async def next(
 
         def _wrapper():
             try:
+                if self._lock is not None and generator_uid not in self._generators:
+                    raise Exception(
```

Comment:
How about raising an RuntimeException since this is totally unexpected？

Reply:
OK